app/listeners/reporting_event_listener.rb

```diff
@@ -16,7 +16,7 @@ def conversation_resolved(event)
       event_start_time: conversation.created_at,
       event_end_time: conversation.updated_at
     )
-    reporting_event.save
+    reporting_event.save!
```

Comment:
On this one I wasn't sure whether the result of this method is checked somewhere.

Reply:
This should be good.